lib/Agrammon/Web/Routes.pm6

```diff
@@ -482,19 +483,24 @@ sub application-routes(Agrammon::Web::Service $ws) {
             }
         }
 
-        # TODO: test/implement load_branch_data()
+        # TODO: test
         post -> LoggedIn $user, 'load_branch_data' {
-            request-body -> (:$name!) {
-                my $data = $ws.load-branch-data($user, $name);
+            request-body -> (:$name!, :%data!) {
+                my $data = $ws.load-branch-data($user, $name, %data);
                 content 'application/json', $data;
             }
         }
 
-        # TODO: test/implement store_branch_data()
+        # TODO: test
         post -> LoggedIn $user, 'store_branch_data' {
-            request-body -> (:datasetName($dataset-name)!, :%data!) {
-                my $data = $ws.store-branch-data($user, %data, $dataset-name);
-                content 'application/json', $data;
+            request-body -> (:name($dataset-name)!, :%data!) {
+                $ws.store-branch-data($user, $dataset-name, %data);
+                CATCH {
+                    note "$_";
```

Comment:
Wanted or debugging leftover?

Reply:
I have those lines in all routes with a `CATCH` so that such failures are recorded in the log file. Is this not a good idea?

Reply:
Ah, right, somehow I thought I was reading this in the DB class; I know that pattern is done here.

Probably Cro can provide something a little nicer for this in the future, but for now it makes sense.